css/homework.css

```diff
@@ -323,4 +520,178 @@ font-family: 'Raleway';
   font-size: 18px;
   width: 270px;
   margin: 0;
+}
+
+.stylish {
+  background-color: rgba(0, 0, 0, 0.5);
+  height: 442px;
+  background-image: url(image/background__stylish.jpg);
+}
+
+.stylish__inner {
+  width: 1170px;
+  margin: auto;
+}
+
+.stylish__title-wrapper {
+  height: 150px;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.stylish__title {
+  font-family: 'Raleway';
```

Comment:
It's better to add font-family and font-weight in section selector instead of doing it in each element.